docs/developer/windows/certificate_store/README.md

```diff
@@ -0,0 +1,143 @@
+# Guide to setting up a Windows Server for Certificate Testing
```

Comment:
Is this also a flow feature? If it is, then should we try adding it to the CI at some point?

Reply:
Sadly almost none of this can be done in powershell, the creating of a certificate can but I couldnt find a way to set the settings I needed. I am doing this in prep for porting it over to Flow. It also requires a server version of windows.